aws/tags.go

```diff
@@ -100,7 +100,17 @@ func setVolumeTags(conn *ec2.EC2, d *schema.ResourceData) error {
 				return nil
 			})
 			if err != nil {
-				return err
```

Comment:
If the error is not a resource timeout error, now the code is completely skipping returning it. 😅 I will fix these on merge.

Reply:
uh, my bad 🙄 thanks!